VTTiny/Components/Animator/AnimatorCharacter.Serialization.cs

```diff
@@ -11,14 +13,23 @@ public partial class AnimatorCharacter : IPackageAble
         /// <returns>The resulting config.</returns>
         public AnimatorCharacterConfig PackageIntoConfig()
         {
+            List<AssetReference<Texture>?>? speakingList = new List<AssetReference<Texture>?>();
```

Comment:
You can safely skip the variable type on declarations like this, and just substitute it with `var`